arches/app/templates/views/components/map-widget-editor.htm

```diff
@@ -2,11 +2,11 @@
 {% load i18n %}
 
 {% block tabs %}
-<div class="workbench-card-sidebar-tab" data-bind="click: function() {
+<div tabindex="0" class="workbench-card-sidebar-tab" data-bind="click: function() {
     toggleTab('editor');
 }, css: {
     'active': activeTab() === 'editor'
-}">
+}" onkeyup="if(event.which == 13 || event.keyCode == 13){ $(this).trigger('click'); }">
```

Comment:
Use the [knockout event binding](https://knockoutjs.com/documentation/event-binding.html) here instead of the DOM event; this applies for other instances of this in this PR as well.

Reply:
@aarongundel These instances of the keyup event should be sorted in this pull request now